api/core/api.proto

```diff
@@ -79,12 +79,12 @@ message EventData {
 // ```
 message ListenResultRequest {
   string serviceID = 1;     // Service ID. Generated when using the `DeployService` API.
-  string taskFilter = 2;    // **Optional.**  Task's key to filter. The task have to match this key. The default is `*` and matches any task.
-  string outputFilter = 3;  // **Optional.** Output's key of the task to filter. The task have to return this output's key. The default is `*` and matches any output.
+  string taskFilter = 2;    // **Optional.**  The task's key to filter. The task must match this key. The default is `*` which matches any task.
+  string outputFilter = 3;  // **Optional.** The output's key from the task to filter. The task must return this output's key. The default is `*` which matches any output.
 }
 
-// Data receive from the stream of the `ListenResult` API.
-// Will be received over time as long as the stream is opened.
+// Data is received from the stream of the `ListenResult` API.
+// It will be received over time as long as the stream is opened.
```

Comment:
Unless I'm not understanding what stream is

Reply:
I think "It will be received over time as long as the stream is open." is perfect.
We use this sentence in 2 places. You fix it for the first place but not the second.